### PR TITLE
Added column with networking type to matrix.

### PR DESCRIPTION
**Pull request by erictune**
@errordeveloper I copied this from your PR #4483.  this will make it easier if you need to rebase  that PR to changes to this file

### CODE REVIEW

**Comment by errordeveloper** [comment]
:+1:

File: docs/getting-started-guides/README.md [hunk 1]
```diff
@@ -1,30 +1,31 @@
...
+Bare-metal     | custom       | Ubuntu | _none_      | [docs](../../docs/getting-started-guides/ubuntu_single_node.md) | Community (@jainvipin)       |
+Bare-metal     | custom       | Ubuntu Cluster | _none_ | [docs](../../docs/getting-started-guides/ubuntu_multinodes_cluster.md) | community (@resouer @WIZARD-CXY) | use k8s version 0.10.1
+Local          |              |        | _none_      | [docs](../../docs/getting-started-guides/locally.md)   | Inactive                     |
+Ovirt          |              |        |             | [docs](../../docs/getting-started-guides/ovirt.md)     | Inactive                     |
+Rackspace      | CoreOS       | CoreOS | Rackspace   | [docs](../../docs/getting-started-guides/rackspace.md) | Inactive                     |
+Bare-metal     | custom       | CentOS | libvirt/KVM | [docs](../../docs/getting-started-guides/centos/centos_manual_config.md) | Community(@coolsvap)    | Uses K8s v0.9.1
+libvirt/KVM    | CoreOS       | CoreOS | Weave       | [docs](../../docs/getting-started-guides/libvirt-coreos.md) | Community (@lhuard1A)   |
```
Comment (errordeveloper) [inline docs/getting-started-guides/README.md]: Not sure how this got mixed up... This one is using the `libvirt/KVM` networking as [I understood][1], which basically implies a bridge on KVM host, and is limited to the VMs on that host.

https://github.com/errordeveloper/kubernetes/blob/a0752db19e2bfc0b825b56d0867331a0e7f9cd9d/docs/getting-started-guides/README.md

Comment (errordeveloper) [inline docs/getting-started-guides/README.md]: - [cluster/libvirt-coreos/util.sh](https://github.com/GoogleCloudPlatform/kubernetes/blob/2c286fea4cd6c79326552acd53d1e6f000e180d3/cluster/libvirt-coreos/util.sh#L156-L159)
- [cluster/libvirt-coreos/network_kubernetes_global.xml](https://github.com/GoogleCloudPlatform/kubernetes/blob/2c286fea4cd6c79326552acd53d1e6f000e180d3/cluster/libvirt-coreos/network_kubernetes_global.xml)
- [cluster/libvirt-coreos/network_kubernetes_pods.xml](https://github.com/GoogleCloudPlatform/kubernetes/blob/2c286fea4cd6c79326552acd53d1e6f000e180d3/cluster/libvirt-coreos/network_kubernetes_pods.xml)

**Comment by erictune** [comment]
good catch, fixed.

**Comment by bgrant0607** [comment]
LGTM

**Pull request merged**